modules/compliance-priorityclass.adoc

```diff
@@ -8,6 +8,10 @@
 
 In large scale environments, the default `PriorityClass` object can be too low to guarantee Pods execute scans on time. For clusters that must maintain compliance or guarantee automated scanning, it is recommended to set the `PriorityClass` variable to ensure the Compliance Operator is always given priority in resource constrained situations.
 
+.Prerequisites
+
+* Optional: Create a `PriorityClass` object. See _Configuring priority and preemption_ in the _Additional Resources_ section for more information.
```

Comment:
This feels like a naive question, but is there a way to link directly? Or is that against convention?

Reply:
Sadly, that is against convention at this time. Long story short, linking from modules breaks the docs build or the webpage or both.